cocos/core/platform/sys.ts

```diff
@@ -670,7 +670,7 @@ export const sys: Record<string, any> = {
      * @en Get the network type of current device, return `sys.NetworkType.LAN` if failure.
      * @zh 获取当前设备的网络类型, 如果网络类型无法获取，默认将返回 `sys.NetworkType.LAN`
      */
-    getNetworkType (): NetworkType {
+    getNetworkType(): NetworkType {
```

Comment:
Do not modify the coding style, please make sure your local eslint setting is synced